src/Roave/SecurityAdvisories/Boundary.php

```diff
@@ -9,8 +9,9 @@
  */
 final class Boundary
 {
-    private const MATCHER = '/^\s*(<|<=|=|>=|>)\s*((?:\d+\.)*\d+)\s*$/';
-    private const VALID_ADJACENCY_MAP = [
+    const STABILITY_TAIL = '[._-]?(?:(stable|beta|b|rc|alpha|a|patch|pl|p)((?:[.-]?\d+)+)?)?([.-]?dev)?';
```

Comment:
If you declared new constants, please make sure they are `private const` unless 100% sure that they should be reused elsewhere

Reply:
sure, my bad. Done!